src/EditorFeatures/CSharpTest/Completion/CompletionProviders/AbstractCSharpCompletionProviderTests.cs

```diff
@@ -133,7 +134,7 @@ protected void VerifySendEnterThroughToEnter(string initialMarkup, string textTy
                 var document = workspace.CurrentSolution.GetDocument(documentId);
                 var position = hostDocument.CursorPosition.Value;
 
-                var completionList = GetCompletionList(document, position, CompletionTriggerInfo.CreateInvokeCompletionTriggerInfo());
+                var completionList = GetCompletionList(document, position, new DisplayListCompletionTrigger());
```

Comment:
as this has no args, why not have a static readonly instance of it?  You do that later on for other triggers it seems lke.